04-wrangling.Rmd

```diff
@@ -337,7 +338,7 @@ summary_temp <- weather %>%
 
 (ref:group-by) Diagram of group_by() and summarize().
```

Comment:
Heads up we can't have dashes in the reference name. Switched to `ref:groupby` in e95810652634b93a74b30d233ec9575aa790b06e